File: amuta/amuta-crm/Seldat.Amuta.Entities/Guarantor.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Seldat.Amuta.Entities
{
   public class Guarantor
    {
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string IdentityNumber { get; set; }
        public string IdCard { get; set; }
        public byte[] DigitalSignature { get; set; }
        public DateTime GuaranteeDeedWritingDate { get; set; }
        public string LoanNote { get; set; }
    }
}

File: amuta/amuta-crm/Seldat.Amuta.Web/src/app/loans/shared/models/guarantor.ts
export class Guarantor {

    constructor(
        public Id:number,
        public FirstName:string,
        public LastName:string,
        public IdentityNumber:string,
        public IdCard:string,
        //public byte[] DigitalSignature ,
        public GuaranteeDeedWritingDate:Date,
        public LoanNote:string,
    ){}

}
